tests/test_meta_functions_against_registry.py

```diff
@@ -9,6 +10,159 @@
 )
 
 
+def test_is_registered(metaregistry, registries, max_pools):
+    for i, registry in enumerate(registries):
+        total_pools = registry.pool_count() if max_pools == 0 else max_pools
+        for pool_index in range(total_pools):
+            pool = registry.pool_list(pool_index)
+            print(pool)
```

Comment:
print statements in tests are uncommon

Reply:
This is from an older commit, `test_meta_functions_against_registry.py` has been deleted and I believe fiddy has refactored a lot of this into `test_getters.py` so this should be gone